Tests/TransferMethod/HyperwalletTransferMethodTests.swift

```diff
@@ -53,32 +53,32 @@ class HyperwalletTransferMethodTests: XCTestCase {
         XCTAssertNotNil(linkNext?.href)
 
         // check the bank card
-        let bankCard = transferMethods?.data.first { ($0.getField(fieldName: .type) as? String ?? "")  == "BANK_CARD" }
-        XCTAssertEqual(bankCard?.getField(fieldName: .type) as? String, "BANK_CARD")
-        XCTAssertEqual(bankCard?.getField(fieldName: .token) as? String, "trm-00002")
+        let bankCard = transferMethods?.data.first { ($0.type ?? "")  == "BANK_CARD" }
+
+        XCTAssertEqual(bankCard?.type, "BANK_CARD")
+        XCTAssertEqual(bankCard?.token, "trm-00002")
+        XCTAssertEqual(bankCard?.status, "ACTIVATED")
+        XCTAssertEqual(bankCard?.transferMethodCountry, "US")
+        XCTAssertEqual(bankCard?.transferMethodCurrency, "USD")
         XCTAssertEqual(bankCard?.getField(fieldName: .cardNumber) as? String, "************1358")
```

Comment:
@jsaini-hw 
As `transferMethods` is of type `HyperwalletPageList<HyperwalletTransferMethod>?`, only instances of base class `HyperwalletTransferMethod` instantiated and attempts to cast down fail.
Leaving as it is for now, will appreciate for a piece of advice.

Reply:
Modify `public func getField(fieldName: TransferMethodField) -> String? {
        return self.storage[fieldName.rawValue]?.value as? String
    }`
Fix all the warnings and leave it as it is that can not be changed to use getters.

Reply:
Thanks, fixed